packages/fast-usdc/src/exos/advancer.js

```diff
@@ -129,14 +133,12 @@ export const prepareAdvancerKit = (
             const advanceAmount = feeTools.calculateAdvance(requestedAmount);
 
             // TODO: consider skipping and using `borrow()`s internal balance check
-            const poolBalance = assetManagerFacet.lookupBalance();
+            const poolBalance = borrowerFacet.getBalance();
             if (!isGTE(poolBalance, requestedAmount)) {
               log(
                 `Insufficient pool funds`,
                 `Requested ${q(advanceAmount)} but only have ${q(poolBalance)}`,
```

Comment:
**note**: I wonder if this makes use of the [ses console](https://github.com/endojs/endo/blob/master/packages/ses/src/error/README.md) as it should.

There's nothing to censor here. But "a more powerful distributed causal console" could be really handy.

Not for this PR, but I wonder when.